go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -616,16 +611,13 @@ func (as *ovnAddressSet) addIPs(ips []net.IP) ([]ovsdb.Operation, error) {
 
 // deleteIPs removes selected IPs from the existing address_set
 func (as *ovnAddressSet) deleteIPs(ips []net.IP) ([]ovsdb.Operation, error) {
-	// dedup
-	uniqIPs := make([]string, 0, len(ips))
```

Comment:
ha its weird we were running this loop and then checking if the list was empty :D